azure_functions_worker/bindings/meta.py

```diff
@@ -25,12 +29,25 @@ def load_binding_registry() -> None:
     global BINDING_REGISTRY
     BINDING_REGISTRY = func.get_binding_registry()
 
-
-def get_binding(bind_name: str) -> object:
-    binding = None
-    registry = BINDING_REGISTRY
-    if registry is not None:
-        binding = registry.get(bind_name)
+    # The SDKs only support python 3.8+
+    if sys.version_info.minor >= BASE_EXT_SUPPORTED_PY_MINOR_VERSION:
+        # Import the base extension
+        try:
+            import azure.functions.extension.base as clients
+            global SDK_BINDING_REGISTRY
+            SDK_BINDING_REGISTRY = clients.get_binding_registry()
+        except ImportError:
+            # This will throw a ModuleNotFoundError in env reload because
+            # azure.functions.extension isn't loaded in
+            pass
+
+
+def get_binding(bind_name: str, pytype: typing.Optional[type] = None) -> object:
+    binding = get_deferred_binding(bind_name=bind_name, pytype=pytype)
+
+    # Either cx didn't import library or didn't define sdk type
+    if binding is None and BINDING_REGISTRY is not None:
+        binding = BINDING_REGISTRY.get(bind_name)
```

Comment:
todo: change order

Reply:
The ordering shouldn't be changed. `bind_name` will be 'blob' for both InputStream and BlobClient types. That means that `BINDING_REGISTRY.get('blob')` will always return a BlobConverter, regardless of the defined type. 

When checking if the binding is a deferred binding first, only one `X_REGISTRY.get()` call will be made. If the check is reversed, a blob trigger with a defined type of BlobClient will call both `BINDING_REGISTRY.get()` and `DEFERRED_BINDINGS_REGISTRY.get()`.